eWonConnector-gateway/src/main/java/org/imdc/ewon/config/EwonConnectorSettings.java

```diff
@@ -17,112 +17,251 @@
 /**
  * The main settings for the eWon connector, containing auth data, and settings that dictate update
  * rates, etc.
- *
```

Comment:
Whitespace change

Reply:
Do you want this in a separate commit? I just did it in this commit, since it pertains to improved code commenting

Reply:
Fixed